manim/scene/scene_file_writer.py

```diff
@@ -725,6 +725,8 @@ def flush_cache_directory(self):
 
     def write_subcaption_file(self):
         """Writes the subcaption file."""
+        if not config["output_file"]:
```

Comment:
I'd prefer a slightly more explicit check:

```suggestion
        if config.output_file is None:
```